src/api/requests/PriceTickerCreateRequest.ts

```diff
@@ -13,16 +13,49 @@ export class PriceTickerCreateRequest extends RequestBody {
     @IsNotEmpty()
     public crypto_symbol: string;
 
+    @IsNotEmpty()
+    public crypto_rank: string;
+
+
+
     @IsNotEmpty()
     public crypto_price_usd: string;
 
     @IsNotEmpty()
     public crypto_price_btc: string;
 
-    @IsNotEmpty()
+
+
+    public crypto_24h_volume_usd: string;
+
+    public crypto_market_cap_usd: string;
+
+    public crypto_available_supply: string;
+
+    public crypto_total_supply: string;
+
+    public crypto_max_supply: string;
+
+
+
+    public crypto_percent_change_1h: string;
+
+    public crypto_percent_change_24h: string;
+
+    public crypto_percent_change_7d: string;
+
+
+    public crypto_last_updated: string;
+
     public crypto_price_currency: string;
 
+    public crypto_24h_volume_currency: string;
+
+    public crypto_market_cap_currency: string;
+
+
+
```

Comment:
adding ?convert=EUR to the request url should give you also prices and volumes in eur